sdcm/cluster.py

```diff
@@ -1165,11 +1165,9 @@ def wait_db_up(self, verbose=True, timeout=3600):
 
     def is_manager_agent_up(self, port=None):
         port = port if port else self.MANAGER_AGENT_PORT
-        if self.is_port_used(port=port, service_name="scylla-manager-agent"):
-            # When the agent is IP, it should answer an https request of https://NODE_IP:10001/ping with status code 204
-            response = requests.get(f"https://{normalize_ipv6_url(self.ip_address)}:{port}/ping", verify=False)
-            return response.status_code == 204
-        return False
+        # When the agent is IP, it should answer an https request of https://NODE_IP:10001/ping with status code 204
+        response = requests.get(f"https://{normalize_ipv6_url(self.ip_address)}:{port}/ping", verify=False)
+        return response.status_code == 204
```

Comment:
what happens if the `get` times out, or you don't get a proper `status_code`?
the previous code covered it...

Reply:
the previous code didn't cover it, if there would be a timeout. it would be raise exception from requests library.

Reply:
wait_for swallows the excepion